code/modules/research/xenoarchaeology/finds/finds_special.dm

```diff
@@ -150,6 +155,37 @@
 		B.blood_DNA[M.dna.unique_enzymes] = M.dna.b_type
 		M.vessel.remove_reagent(/singleton/reagent/blood,rand(25,50))
 
+/obj/item/vampiric/proc/bloodcall_final(var/mob/living/carbon/human/M)
+	if(istype(M))
+		if(istype(M.species, /datum/species/machine))
+
+			M.apply_damage(700, DAMAGE_BRUTE, BP_GROIN)
+			M.apply_damage(700, DAMAGE_BRUTE, BP_GROIN) //The way IPC limbs seem to act I needed the double damage to ensure it was removed.
+			to_chat(M, SPAN_HIGHDANGER("What have you done?! You feel like you're being ripped in half!"))
+			return
+
+		to_chat(M, SPAN_HIGHDANGER("What have you done?! You feel an incredible pull as the statue breaks. The force crushes your hands and pulls your blood into an unseen void."))
+		M.apply_damage(30, DAMAGE_BRUTE, BP_L_HAND)
+		M.apply_damage(30, DAMAGE_BRUTE, BP_R_HAND)
+		M.vessel.remove_reagent(/singleton/reagent/blood,rand(120,175))
+
+/obj/item/vampiric/proc/shatter(var/obj/item/W, var/mob/user)
+	playsound(src, shatter_sound, 70, 1)
+	audible_message(SPAN_WARNING("\The [src] shatters with a resounding crash!"), SPAN_WARNING("\The [src] breaks."))
+	playsound(src.loc, pick('sound/hallucinations/wail.ogg','sound/hallucinations/veryfar_noise.ogg','sound/hallucinations/far_noise.ogg'), 200, 1, -3)
```

Comment:
```suggestion
	playsound(get_turf(src), pick('sound/hallucinations/wail.ogg','sound/hallucinations/veryfar_noise.ogg','sound/hallucinations/far_noise.ogg'), 200, TRUE, -3)
```
Also, -3 isn't acceptable there, use one of the `*_SOUND_EXTRARANGE` defines in `code\__DEFINES\sound.dm` (you can offset them, but it must be based on it)

I am not sold on the volume being 200 either, but I'm not sure that's an error

Reply:
Changes applied. I've run multiple tests using the 200 volume and haven't seen any advserse effects but if there is some reason to lower or change it let me know.